ec2/spark_ec2.py

```diff
@@ -1082,11 +1109,11 @@ def real_main():
                     time.sleep(30)  # Yes, it does have to be this long :-(
                     for group in groups:
                         try:
-                            conn.delete_security_group(group.name)
-                            print "Deleted security group " + group.name
+                            conn.delete_security_group(group_id=group.id)
+                            print "Deleted security group %s, id %s" % (group.name, group.id)
```

Comment:
Is there any place where the user inputs a group by its ID instead of its name? If not, I don't think we should print this here.